tests/monitoring/monitoring_service/test_end_to_end.py

```diff
@@ -66,29 +113,124 @@ def test_e2e(
 
     ms_greenlet.kill()
     # c1 asks MS to monitor the channel
-    # reward_amount = 1
-    # monitor_request = c1.get_monitor_request(
-    #     c2.address,
-    #     balance_proof_c2,
-    #     reward_amount,
-    #     monitoring_service.address,
-    # )
-    # # wait for channel open event to be processed by the MS
-    # wait_for_blocks(1)
-    # gevent.sleep(TEST_POLL_INTERVAL)
+    reward_amount = 1
+    monitor_request: raiden_libs.messages.MonitorRequest = c1.get_monitor_request(
+        c2.address,
+        balance_proof_c2,
+        reward_amount,
+        monitoring_service.address,
+    )
+    # wait for channel open event to be processed by the MS
+    wait_for_blocks(1)
+    gevent.sleep(0)
+
+    mr = MonitorRequest(
```

Comment:
This would be much shorter if we could use the same MR object.

Reply:
True, now I can fix this directly in the `RaidenMockClient` class.